Added code in order to support "search for owner criteria"

Miquilin opened the pull request:
changed the flow of menu, search for owner is only displayed in windows os.
Added code on search side in order to import the win32 library and use the function related only on windows os

mtancara committed:
Added code i order tosupport search for owner criteria

changed the flow of menu, search for owner is only displayed in windows os.
Added code on search side in order to import the win32 library and use the function related only on windows os

ovandononi reviewed (changes requested):
Observation 1: Menu flow requires twice that an user fills out owner criteria search
Observation 2: Unit test about owner criteria search should updated with windows OS condition similar than updated code on search py

mtancara committed:
Fixed the problem in  the advanced flow menu

Fixed the problem in the flow where the owner criteria menu is being displayed 2 times

ovandononi reviewed (approved):
appoved
